src/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -403,7 +410,7 @@ export class MongoEntityConverter {
         return LookaheadMode.WHEN_CLEAR
       }
       default: {
-        console.log(`### Warning: Found unknown number for LookAhead: ${lookAheadNumber}`)
+        this.logger.warn(`Found unknown number for LookAhead: ${lookAheadNumber}`)
         // TODO: Throw error. Currently we have some misconfiguration that uses an outdated lookAhead mode
         // throw new UnsupportedOperation(`Found unknown number for LookAhead: ${lookAheadNumber}`)
```

Comment:
I'm tempted to say that we should just let it throw the error now intead of logging. That way we don't need to parse in a logger to this file and we can remove the misconfigguration from the instances that have it.

Reply:
Throws the exception now.